src/log/log.c

```diff
@@ -2090,7 +2090,21 @@ __wt_log_scan(WT_SESSION_IMPL *session, WT_LSN *lsnp, uint32_t flags,
     if (log != NULL) {
         allocsize = log->allocsize;
         WT_ASSIGN_LSN(&end_lsn, &log->alloc_lsn);
-        WT_ASSIGN_LSN(&start_lsn, &log->first_lsn);
+        if (start_range_lsnp != NULL) {
```

Comment:
Just to make sure I'm understanding correctly the reason we're not using `lsnp` is that `start_range_lsnp` is giving us an approximate starting point whereas `lsnp` has to exist?

Reply:
Yeah! I also didn't want to change anything to do with lsnp since it's relied upon by other calls to __wt_log_scan